schemas/predefined_plans.go

```diff
@@ -0,0 +1,98 @@
+package schemas
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/Peripli/service-manager/pkg/instance_sharing"
+	"github.com/Peripli/service-manager/pkg/types"
+	"github.com/gofrs/uuid"
+	"time"
+)
+
+func BuildReferencePlanSchema() string {
+	return fmt.Sprintf(`{
+  "name": "%[1]s",
+  "catalog_name": "%[1]s",
+  "description": "%[2]s",
+  "bindable": true,
+  "ready": true,
+  "metadata": {
+    "supportedPlatforms": []
+  },
+  "schemas": {
+    "service_instance": {
+      "create": {
+        "parameters": {
+          "$schema": "http://json-schema.org/draft-04/schema#",
+          "type": "object",
+          "additionalProperties": false,
+          "_show_form_view": true,
+		  "_controlsOrder": ["%[3]s", "%[6]s", "%[9]s", "%[12]s"]
+          "properties": {
+            "%[3]s": {
+              "title": "%[4]s",
+              "description": "%[5]s",
+              "type": "string",
+              "minLength": 0,
+              "maxLength": 100
+            },
+            "%[6]s": {
+              "title": "%[7]s",
+              "description": "%[8]s",
+              "type": "string",
+              "minLength": 0,
+              "maxLength": 100
+            },
+            "%[9]s": {
+              "title": "%[10]s",
+              "description": "%[11]s",
+              "type": "string",
+              "minLength": 0,
+              "maxLength": 100
+            },
+            "%[12]s": {
+              "title": "%[13]s",
+              "description": "%[14]s",
+              "type": "object"
+            }
```

Comment:
Waiting for uriel.even.sapir@sap.com review, to verify we can use an object input without properties.